### PR TITLE
NXP backend: Add documentation for channels last dim order support in Neutron backend + example in `aot_neutron_compile.py`.

### PR DESCRIPTION
**Pull request by MartinPavella**
### Summary
This PR adds a markdown documentation for the channels last dim order support from the NXP backend, and an example use in the `aot_neutron_compile.py`.

### Test plan
N/A

cc @robert-kalmar

### CODE REVIEW

File: examples/nxp/aot_neutron_compile.py [hunk 1]
```diff
@@ -206,6 +214,15 @@ def get_model_and_inputs_from_name(model_name: str):
     )
     model = model.eval()
 
+    if args.use_channels_last_dim_order:
+        # Turn the model to channels last.
+        model.to(memory_format=torch.channels_last)
```
Comment (jirioc) [inline examples/nxp/aot_neutron_compile.py:219]: Probably the assignment is missing:  `model = model.to(...)`

Comment (MartinPavella) [reply examples/nxp/aot_neutron_compile.py:219]: It is actually not necessary. The call `.to()` modifies the original instance.

**Review by jirioc — APPROVED** [review]
